README.md

```diff
@@ -49,55 +50,65 @@ pip install git+https://github.com/keras-team/keras-cv.git tensorflow --upgrade
 ## Quickstart
 
 ```python
-import keras_cv
 import tensorflow as tf
 from tensorflow import keras
+from keras_cv.layers import RandomFlip, RandAugment, CutMix, Mixup
+from keras_cv.models import ResNet50V2Backbone, ImageClassifier
 import tensorflow_datasets as tfds
 
-# Create a preprocessing pipeline
+# Create a preprocessing pipeline with augmentations
+BATCH_SIZE = 16
+NUM_CLASSES = 3
 augmenter = keras.Sequential(
-    layers=[
-        keras_cv.layers.RandomFlip(),
-        keras_cv.layers.RandAugment(value_range=(0, 255)),
-        keras_cv.layers.CutMix(),
-        keras_cv.layers.MixUp()
+    [
+        RandomFlip(),
+        RandAugment(value_range=(0, 255)),
+        CutMix(),
+        MixUp()
```

Comment:
Feels a bit sceptical to use these two heavy transformation sequentially. I wonder if there's any literature explore this area. Otherwise what if showcase the random_choice layer on this two, no?

```python
layers.RandomChoice(
   layers=[CutMix(), MixUp(), FourierMix()]
)
```

Reply:
Great we're just waiting for #1749 @LukeWood